### PR TITLE
Dashgraph

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review .. before merging this one only to development , pls push the current development branch to master

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,600,800" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
   <link rel="stylesheet" href="https://unpkg.com/react-rangeslider/umd/rangeslider.min.css" />
-
+  <script src="https://www.gstatic.com/charts/loader.js"></script>
```
Comment (ShankyRana) [inline public/index.html]: isn't there a package for this?

File: src/components/WebsiteContact/WebsiteContact.js [hunk 2]
```diff
@@ -116,12 +155,17 @@ class WebsiteContact extends Component {
                       placeholder="Leave us a message"
                       className="validate-required"
                       value={message}
-                      onChange={(e) => this.handleStateChange(e.target.name, e.target.value)}
+                      onBlur={this.checkMsgBlur}  onChange={(e) => this.handleStateChange(e.target.name, e.target.value)}
```
Comment (ShankyRana) [inline src/components/WebsiteContact/WebsiteContact.js]: please keep the alignments of the code inplace here also wherever you see.
Keep code aligned and readable.

File: public/index.html (hunk 1 above)
Comment (Devilla) [reply public/index.html]: yeah we've but Ray's saying it was required as per its documentation. Not working without this.
It's required.

Comment (GAUTAMRAJU15) [reply public/index.html]: in the old graph module we dont have GeoChart that is the world map graph . it's available in the new module we have imported

**Pull request closed without merging**